README.md

```diff
@@ -22,9 +22,11 @@ portals:
 4. Run `npm run setup` to create the custom object
 5. Run `npm run upload` to upload the `hub-homes` theme
 6. Add your API key by running `hs secrets add APIKEY <apikey>`
-7. Create a form that contains the following fields:
+7. [Create a form](https://app.hubspot.com/l/forms) that contains the following fields:
 - First Name
 - Last Name
 - Email Address
-8. Create a page that will display all property listings by using the "Home page" (`property-listings.html`) template
-9. Create a page that will display property details by using the "Details page" (`details.html`) template. The slug for for Details page must be `/property`; you can change the content slug under the page's settings. Additionally, in the "Property Details Module", you'll need to choose the form that you created in step 7.
+8. Upload the `hub-homes` folder from the repo to your portal by running `hs upload ./hub-homes hub-homes`
```

Comment:
Step 5 should have this covered

Reply:
Weird, it didn't work for me then. I can run through it again, but when I went to create a page I didn't see the template there.

Reply:
![image](https://user-images.githubusercontent.com/6472448/94717399-4535fb80-031e-11eb-8450-0ebd1733fcf2.png)

I will add `&& npm run upload` to the setup script as it wasn't there and remove this step from the docs.

Reply:
I think there was a reason that @drewjenkins split those up originally

Reply:
I think keeping `upload` separate is important because it is also useful in the flow outside of the initial setup. My take is that we need to find the right balance between number of steps and it being clear what is happening during each step since a part of this project is as an educational tool.

Reply:
Agree ^ It should be known that they are actually modifying files on their site